src/app/pages/partners/partner-details/partner-details.scss

```diff
@@ -24,10 +24,10 @@
     .scrolling-region {
         background-color: ui-color(white);
         height: 100%;
-        max-width: 100%;
+        max-width: $content-max;
```

Comment:
I don't see $content-max var defined anywhere but it's probably a global

Reply:
It's in the pattern library.